rxros_listener/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rxros_listener)
+
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+endif()
```

Comment:
This is actually only supported starting with CMake 3.x (not sure which version exactly) I believe.

Minimum version we spec right now is `2.8.3`.

Reply:
I read this as: let's leave it as is for now then.